consensus/buildtest.go

```diff
@@ -32,7 +31,7 @@ var (
 
 	CoinbaseAugment = new(big.Int).Lsh(big.NewInt(1), 80)
```

Comment:
oh. no more!

instead of 2^80, I want to use 10^24 (makes it easier to talk in SI notation).